network/transport/nats/nats_transport.go

```diff
@@ -0,0 +1,30 @@
+package nats
+
+import (
+	"github.com/nats-io/nats.go"
+	"github.com/statechannels/go-nitro/network/transport"
+)
+
+type natsTransport struct {
+	connection transport.Connection
+}
+
+var _ transport.Transport = (*natsTransport)(nil)
+
+func NewNatsTransport(nc *nats.Conn, pubTopicNames []string) *natsTransport {
+	// wouldn't it be better to get messaged directly into the channel?
```

Comment:
Can we resolve this now?

Reply:
I am actually not 100% sure what this means. Maybe @calj can elaborate.